app/views/users/index.html.erb

```diff
@@ -1,36 +1,15 @@
 <h1>User Account Management</h1>
 <div class="row" style="padding-top: 10px; padding-bottom: 20px;">
-  <div class="col-md-8">
+  <div class="col-md-12">
     <%= link_to "Add New User", new_user_path, class: "btn btn-primary" %>
-  </div>
-  <div class="col-md-4">
+    <%= link_to "Reset Password (WIP)", "#", class: "btn btn-primary" %>
+    <button type="button" class="btn btn-primary">Toggle Lock (WIP)</button>
     <div style="float: right;">
-      <%= bootstrap_form_tag url: '/search', remote: true, layout: :inline do |p| %>
-        <%= p.text_field :search, placeholder: 'search', hide_label: true, class: 'search_form_input' %>
-        <%= p.submit 'Search', class: 'btn btn-primary' %>
-      <% end %>
+      <%= render partial: 'users/search_form', locals: { autosortable: true } %>
```

Comment:
Thing to add as an issue: the stupidtable sort doesn't work after a search. Let's fix in a separate PR.